sdk-extensions/jaeger-remote-sampler/build.gradle.kts

```diff
@@ -3,6 +3,8 @@ plugins {
   id("otel.publish-conventions")
 
   id("otel.animalsniffer-conventions")
+
+  id("com.squareup.wire")
 }
 
```

Comment:
I don't think we need as many as the OTLP tests but can we add one test suite with the grpc dependencies used?

https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/otlp/trace/build.gradle.kts#L31

It's ok to just copy the test code instead of abstract

Reply:
I have added support only for "grpc light" approach via okhttp.